src/docfx/validation/MetadataValidator.cs

```diff
@@ -52,14 +52,8 @@ public void ValidateMetadata(ErrorBuilder errors, JObject metadata, Document fil
 
             foreach (var schemaValidator in _schemaValidators)
             {
-                if (file.PageType != null)
+                if (file.PageType != null && file.PageType != "toc")
```

Comment:
remove this hard code, every valdiation rules should have contentTypes

Reply:
done